Taxi/Taxi/Presenter/AddTaxiParty/AddTaxiParty.swift

```diff
@@ -42,7 +42,252 @@ struct AddTaxiParty: View {
     }
 
     var body: some View {
-        Text(/*@START_MENU_TOKEN@*/"Hello, World!"/*@END_MENU_TOKEN@*/)
+        VStack(alignment: .leading, spacing: 0) {
+            closeButton
+            destinationSelect
+            datePicker
+                .offset(y: -1)
+            timePicker
+                .offset(y: -2)
+            departurePicker
+                .offset(y: -3)
+            personNumberPicker
+                .offset(y: -4)
+            Spacer()
+            if checkAllInfoSelected() {
+                guideText
+            }
+            RoundedButton("택시팟 생성", !checkAllInfoSelected()) {
+                // TODO: 택시팟 생성 유즈케이스와 연결하기!
+            }
+        }
+    }
+
+    private var guideText: some View {
+        HStack {
+            Spacer()
+            Text("해당 사항이 맞으시면 택시팟 생성을 눌러주세요")
+                .caption()
+                .padding(.bottom, 12)
+            Spacer()
+        }
+    }
+
+    private var closeButton: some View {
+        Button {
+            dismiss()
+        } label: {
+            Image(systemName: "xmark")
+                .imageScale(.large)
+                .padding()
+        }
+    }
+}
+// MARK: - 여기서부턴 목적지를 설정하는 뷰입니다.
+extension AddTaxiParty {
+    private var destinationSelect: some View {
+        HStack(spacing: 16) {
+            ForEach(Place.destinations(), id: \.self) { destination in
+                destinationItem(destination)
+            }
+        }
+        .padding(EdgeInsets(top: 24, leading: 16, bottom: 24, trailing: 16))
+        .background(Color.addBackground)
+        .toInfoContainer(title: "어디로 가시나요?", selectedInfo: destination?.rawValue ?? "", toggle: step == .destination) {
+            changeStep(to: .destination)
+        }
+    }
+
+    private func destinationItem(_ place: Place) -> some View {
+        Button {
+            if destination != place {
+                destination = place
+                departure = nil
+                toNextStep()
+            }
+        } label: {
+            Text(place.rawValue)
+                .info()
+                .padding()
+                .frame(maxWidth: .infinity)
+                .roundedBackground(destination == place)
+        }
+    }
+}
+// MARK: - 날짜 선택 뷰
+extension AddTaxiParty {
+    private var datePicker: some View {
+        Button {
+            startDate = "1월 1일"
+            toNextStep()
+        } label: {
+            Text("추후 구현 예정")
+                .padding()
+        }
+        .toInfoContainer(title: "날짜를 선택해주세요", selectedInfo: startDate ?? "", toggle: step == .date) {
+            changeStep(to: .date)
+        }
+    }
+}
+// MARK: - 시간 선택 뷰
+extension AddTaxiParty {
```

Comment:
```suggestion
private extension AddTaxiParty { 
```

extension에 private 가 설정되면 exntension에서 정의된 method들도 자동으로 private이 된답니다 : )

Reply:
오호 그러면 private extension 내부에 선언된 함수들에는 명시적으로 접근 지정자를 붙이지 않아도 되는거군요!?🥹

Reply:
8b863738e518ed261bb4f20a1dc3a61c62879a2f 에 반영했습니다!